Inc/config.h

```diff
@@ -0,0 +1,80 @@
+/* include/config.h.  Generated from config.h.in by configure.  */
```

Comment:
How come we didn't have a config.h before? Is this specifically for yaml? If so, should we rename it something more specific?

Reply:
I've renamed it and moved it to the yaml folder.